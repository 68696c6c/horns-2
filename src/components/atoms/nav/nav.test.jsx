/* global describe, it, expect */
import React from 'react'

import { render } from 'setup-test'

import Nav from '.'

describe('Nav', () => {
  it('should render as default', () => {
    const { container } = render(
      <Nav>
        <a href="/one">One</a>
        <a href="/two">Two</a>
        <a href="/three">Three</a>
      </Nav>
    )
    expect(container).toMatchSnapshot()
  })
})
