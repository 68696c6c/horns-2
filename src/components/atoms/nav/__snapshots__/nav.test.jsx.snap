// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav should render as default 1`] = `
@media (min-width:1200px) {
  .emotion-0 {
    padding-left: calc(((100vw - 1200px) / 2));
    padding-right: calc(((100vw - 1200px) / 2));
  }
}

<div>
  <nav
    class="emotion-0"
  >
    <a
      href="/one"
    >
      One
    </a>
    <a
      href="/two"
    >
      Two
    </a>
    <a
      href="/three"
    >
      Three
    </a>
  </nav>
</div>
`;
